package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "photo-gallery-module",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "react-dev": "webpack -d --watch",
+    "start": "nodemon ./server/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/leokyeung/photo-gallery-module.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/leokyeung/photo-gallery-module/issues"
+  },
+  "homepage": "https://github.com/leokyeung/photo-gallery-module#readme",
+  "dependencies": {
+    "eslint": "^6.7.2",
+    "express": "^4.17.1",
+    "mysql": "^2.17.1",
```

Comment:
In the sequelize getting started doc, they used "mysql2" not "mysql" but I think don't think it should have any problems? What do you think?

Reply:
Thanks for pointing it out. I will follow the doc for best practice.